src/modules/Microsoft.FinOpsHubs/linkedservices/deploy.bicep

```diff
@@ -0,0 +1,75 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Conditional. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('The name of the service to link to.')
+param linkedServiceName string
+
+@allowed([
+  'AzureBlobFS'
+  'AzureKeyVault'
+])
+@description('Required. The type of Linked Service.')
+param linkedServiceType string
+
+@description('The name of the service to link to.')
+param storageAccountName string = 'none'
```

Comment:
Should storageAccountName be conditionally required if `linkedServiceType == 'AzureBlobFS'`?